source/_static/mytheme.css

```diff
@@ -2759,7 +2759,9 @@ mark {
 #install-mattermost-server-using-the-tarball p code,
 #install-mattermost-server-using-the-tarball h3 code,
 #install-mattermost-server-on-ubuntu p code,
-#install-mattermost-server-on-ubuntu h3 code {
+#install-mattermost-server-on-ubuntu h3 code,
+#install-mattermost-on-rhel p code,
+#install-mattermost-on-rhel h3 code {
```

Comment:
@cwarnermm Re: inline code - every page has a unique ID on the content container. 
<img width="578" alt="Screen Shot 2023-09-01 at 10 35 18 AM" src="https://github.com/mattermost/docs/assets/10912003/a7e0a3e0-15e8-4b8b-9998-2dc7205891a9">

What you'll want to do is add that id to the list of selectors we have for the inline code on install pages. Specifically, for paragraph and h3 elements on the page (as that's where we're using them thus far).
```css
#unique-id-here p code,
#unique-id-here h3 code 
```

We can probably scale this via some sort of logic esp if we keep adding this styling to more pages, but we can do that in another pass.

Lemme know, if anything - thanks!

Reply:
Thank you so much, @emdecr!